lib/output.js

```diff
@@ -67,6 +71,14 @@ module.exports = function (parameters, response) {
     threshold
   ));
 
+  if (parameters.or) {
+    console.log('\nHere are your optimized images: ', optimizedResoruceURL);
+  }
+
+  if (parameters.download) {
+    got.stream(optimizedResoruceURL).pipe(createWriteStream('optimized.zip'));
+  }
```

Comment:
It's an async function and it's in a sync function. You'll need to convert the function to an async one with callback.